src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +14,26 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private final String regex = "\\W";
+    private final String letter = "w";
+    private final String space = " ";
+    private final String emptyString = "";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        final Path path = Paths.get(fileName);
+        try {
+            if (Files.exists(path)
+                    && Files.isReadable(path)
+                    && Files.size(path) > 0) {
+                return Arrays.stream(Files.readString(path).split(space))
```

Comment:
you are splitting by " " here
do words separated by "\n" get split?

Reply:
yes

Reply:
unfortunately no
`Files.readString(path)` - here you get one big line
and line breaks look the following  - `\n'

when you do `.split(" ") then, the words separated with `\n' remain together

Reply:
I've just tried to split string line which contains several ```\n``` and ```" "```  by ```.split(" ")``` and it was splitted everywhere...

Reply:
Do you mean that I need to use ```\\s+``` instead ```" "```? I've found it and changed